src/main/resources/com/ncc/aif/version.properties

```diff
@@ -0,0 +1 @@
+version=${project.version}
```

Comment:
It looks like these properties files are missing a newline.

Reply:
Good catch. Fixed